src/Sarif.UnitTests/SarifTransformerTests.cs

```diff
@@ -463,5 +463,345 @@ public void SarifTransformerTests_ToCurrent_OneRunWithBasicInvocation()
 
             v2LogText.Should().Be(v2LogExpectedText);
         }
+
+        [Fact]
+        public void SarifTransformerTests_ToCurrent_OneRunWithInvocationAndNotifications()
+        {
+            string v1LogText =
+              @"{
+                  ""version"": ""1.0.0"",
+                  ""runs"": [
+                    {
+                      ""invocation"": {
+                        ""commandLine"": ""CodeScanner @collections.rsp"",
+                        ""responseFiles"": {
+                          ""collections.rsp"": ""-input src/collections/*.cpp -log out/collections.sarif -rules all -disable C9999""
+                        },
+                        ""startTime"": ""2016-07-16T14:18:25Z"",
```

Comment:
Since this test focuses on notifications, and we have another test that exhaustively tests the transfer of these properties, you don't have to repeat them here. Check the schema; I'm sure they're all optional.

Just a single property like `"commandLine"` would suffice, just to show that you can transfer _both_ the notifications and the loose properties.

Reply:
Done